src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import ClickModal from './ClickModal';
+import Modal from './Modal';
+
+export {ClickModal};
+export default Modal;
```

Comment:
How should these imports be handled?

1. Move the `ClickModal` to its own component folder?
2. Have two named exports?
3. Decide on which one is the more "default" one, and the other is the named export?
4. Is there an actual need for both kind of modals? Could imagine that we always only open the modals when an element is clicked.